platforms/android/android.toolchain.cmake

```diff
@@ -711,6 +712,14 @@ elseif( ANDROID_ABI STREQUAL "armeabi-v7a with NEON" )
  set( CMAKE_SYSTEM_PROCESSOR "armv7-a" )
  set( VFPV3 true )
  set( NEON true )
+elseif( ANDROID_ABI STREQUAL "armeabi-v7a-hard with NEON" )
+ set( ARMEABI_V7A_HARD true )
+ set( ANDROID_NDK_ABI_NAME "armeabi-v7a-hard" )
```

Comment:
Ahh that could be. I use buck to build my APK and dont build the samples so didn't check on that. But maybe worth getting this in and iterating on it so that we can start fixing the various hardfp failures